test/evergreen.yml

```diff
@@ -3598,15 +3598,15 @@ tasks:
     #        Performance Long Tests           #
     ###########################################
 
-  - name: perf-test-long-btree
+  - name: perf-test-500m-btree-populate
```

Comment:
Can you update these tests to start with `perf-test-long` again? It makes it easier to group them in the Evergreen scheduling interface.

Reply:
Thanks 😄